com.handtrixxx.OpenCloud.yml

```diff
@@ -0,0 +1,157 @@
+id: com.handtrixxx.OpenCloud
```

Comment:
if this is the official client why isn't this app id related to an opencloud domain ?